src/inc/eventtracebase.h

```diff
@@ -216,11 +280,21 @@ extern "C" {
 #endif //!DONOT_DEFINE_ETW_CALLBACK && !DACCESS_COMPILE
 
 #include <clretwallmain.h>
+
+#elif defined(PLATFORM_UNIX)
```

Comment:
Actually PLATFORM_LINUX is not defined yet since FEATURE_EVENT_TRACE is used on the feature, we should be safe till we enable MAC